### ISSUE TITLE
[T422-047] Importing private type with Default_Initial_Condition via Address aspect causes bugbox

### ISSUE THREAD

**Issue by jklmnn** [Componolit/Workarounds#21]
### Issue

If type `T` is defined as
```Ada
type T is private with
   Default_Initial_Condition;
```
GNAT will get a bugbox if it is used as
```Ada
Var : T with
   Import,
   Address => Adr;
```

### Workaround

Define a private constant for `T` that is used for initialization and remove `Default_Initial_Condition`.

**Comment by treiher** [comment]
As far as I know there is no Pro 21.0 yet. Did you mean Pro 21.0w? I'm not sure if we want to track wavefronts, though.

**Comment by jklmnn** [comment]
Yes I meant Pro 21.0w. I don't we also should track wavefronts. We just should change the labels when it has been fixed in a wavefront.

**Comment by jklmnn** [comment]
Currently they have a fix in a wavefront so I'd assume it will be fixed in Pro 21.0 so should we also add a fixed label even though we don't have a version with a fix yet?

**Comment by treiher** [comment]
Yes, at least that is how I have been handling it so far.
